src/main/scala/fr/polytechnique/cmap/cnam/etl/transformers/follow_up/FollowUpTransformer.scala

```diff
@@ -9,10 +9,25 @@ import org.apache.spark.sql.Dataset
 import fr.polytechnique.cmap.cnam.etl.events._
 import fr.polytechnique.cmap.cnam.etl.patients.Patient
 
-
+/** It allows create a followUp dataset using the dataset of
+ * Dataset[(Patient, Event[ObservationPeriod])],
+ * Dataset[Event[Molecule]] ( This dataset is not used in the treatment and it should be removed in further versions),
+ * Dataset[Event[Outcome]]( This dataset is not used in the treatment and it should be removed in further versions) and
+ * Dataset[Event[Trackloss]].
+ *
+ * @param config A config file tha contains the need values to set the parameters of study.
+ */
 class FollowUpTransformer(config: FollowUpTransformerConfig) {
 
-
+  /** The main method in this transformation class, it allows the transformation into a FollowUp dataset.
```

Comment:
better to frame it "It combines multiple basic Events to form a FollowUp Dataset."